src/valkey-benchmark.c

```diff
@@ -292,16 +292,16 @@ static redisContext *getRedisContext(const char *ip, int port,
 
 
 
-static redisConfig *getRedisConfig(const char *ip, int port,
+static serverConfig *getRedisConfig(const char *ip, int port,
```

Comment:
How about changing `getRedisConfig` also? It seems related to this PR and it's local in this file only.

Reply:
well done